webapi/src/test/scala/org/knora/webapi/routing/AuthenticatorSpec.scala

```diff
@@ -175,5 +175,11 @@ class AuthenticatorSpec extends CoreSpec("AuthenticationTestSystem") with Implic
       }
 
     }
+
+    "called, the 'calculateCookieName' method" should {
+      "succeed with generating the name" in {
+        Authenticator.calculateCookieName(settings) should equal("KnoraAuthenticationGAXDALRQFYYDUMZTGMZQ9999")
+      }
```

Comment:
is there an unhappy path that you could test? (if you made a copy of the settings where the `.externalKnoraApiHostPort` is an empty string or an invalid value, you'd expect a certain error, right?)